website/static/js/registerNode.js

```diff
@@ -13,27 +13,14 @@ var preRegisterMessage =  function(title, parentTitle, parentUrl, category) {
             '</b> including all components and data within it. This will <b>not</b> register' +
             ' its parent, <b>' + parentTitle + '</b>.' +
             ' If you want to register the parent, please go <a href="' +
-            parentUrl + '">here.</a>' +
-            // TODO(hrybacki): Remove once Retraction/Embargoes goes is merged into production
-            '<hr /><b>Important Note:</b> As early as <u>June 8, 2015</u>, registrations ' +
-            'will be made public immediately or can be embargoed for up to four years. ' +
-            'There will no longer be the option of creating a permanently private ' +
-            'registration. If you register before June 8, 2015 and leave your ' +
-            'registration private, then the registration can remain private. After June 8, 2015, ' +
-            'if you ever make it public, you will not be able to return it to private. ';
+            parentUrl + '">here.</a> After selecting OK, you will next select a registration form.';
```

Comment:
Some users have commented that they were nervous to move ahead in registration because they weren't sure when they were committing to the irreversible step.  Just a quick line to reassure them that OK doesn't complete the registration.